LICENSE

```diff
@@ -1,53 +1,62 @@
-                    GNU GENERAL PUBLIC LICENSE
-                       Version 3, 29 June 2007
-
- Copyright (C) 2007 Free Software Foundation, Inc. <http://fsf.org/>
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
```

Comment:
What is bad on plaintext?

Reply:
The window looked bad, asymmetric and the text didn't resize along with it.